frontend/src/components/grantaccess/form/RoleSelect.vue

```diff
@@ -2,13 +2,12 @@
 import { computed } from 'vue';
 import { ErrorMessage, Field } from 'vee-validate';
 import Dropdown from 'primevue/dropdown';
-import type { FamApplicationRole } from 'fam-app-acsctl-api';
 import type { FamRoleDto } from 'fam-admin-mgmt-api/model';
 
 const props = defineProps({
     roleId: { type: Number, default: 0 },
     roleOptions: {
-        type: [Array<FamApplicationRole>, Array<FamRoleDto>],
+        type: [Array<FamRoleDto>],
```

Comment:
Check the return of the get_admin_user_access api, the role type is FamRoleDto, so the FamApplicationRole should be the previous type

<img width="629" alt="Screen Shot 2024-05-21 at 2 09 33 PM" src="https://github.com/bcgov/nr-forests-access-management/assets/23131735/762a8ad7-11e5-4b2c-ae96-74412080bf59">